evalai/get_token.py

```diff
@@ -29,4 +29,8 @@ def get_token():
                 tokendata = json.loads(data)
                 echo("Current token is {}".format(tokendata["token"]))
             except (OSError, IOError) as e:
-                echo(e)
+                echo(str(e))
+            except JSONDecodeError as e:
+                echo("Token file was found empty, Please set the token using the command evalai set_token <token>")
```

Comment:
@aaaakshat can you replace `Please` with `please`

Reply:
done.